features/scaffold-package.feature

```diff
@@ -3,66 +3,96 @@ Feature: Scaffold WP-CLI commands
   Scenario: Scaffold a WP-CLI command without tests
     Given an empty directory
 
-    When I run `WP_CLI_PACKAGES_DIR=packages wp scaffold package wp-cli/foo --skip-tests`
+    When I run `wp package path`
+    Then save STDOUT as {PACKAGE_PATH}
+
+    When I run `wp scaffold package wp-cli/foo --skip-tests`
     Then STDOUT should contain:
       """
       Success: Created package files
       """
-    And the packages/vendor/wp-cli/foo/.gitignore file should exist
-    And the packages/vendor/wp-cli/foo/.editorconfig file should exist
-    And the packages/vendor/wp-cli/foo/.distignore file should exist
-    And the packages/vendor/wp-cli/foo/.distignore file should contain:
+    And the {PACKAGE_PATH}/local/wp-cli/foo/.gitignore file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/foo/.editorconfig file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/foo/.distignore file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/foo/.distignore file should contain:
       """
       .gitignore
       """
-    And the packages/vendor/wp-cli/foo/composer.json file should exist
-    And the packages/vendor/wp-cli/foo/composer.json file should contain:
+    And the {PACKAGE_PATH}/local/wp-cli/foo/composer.json file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/foo/composer.json file should contain:
       """
       "type": "wp-cli-package",
       """
-    And the packages/vendor/wp-cli/foo/composer.json file should contain:
+    And the {PACKAGE_PATH}/local/wp-cli/foo/composer.json file should contain:
       """
       "homepage": "https://github.com/wp-cli/foo",
       """
-    And the packages/vendor/wp-cli/foo/composer.json file should contain:
+    And the {PACKAGE_PATH}/local/wp-cli/foo/composer.json file should contain:
       """
       "license": "MIT",
       """
-    And the packages/vendor/wp-cli/foo/composer.json file should contain:
+    And the {PACKAGE_PATH}/local/wp-cli/foo/composer.json file should contain:
       """
           "require": {
               "wp-cli/wp-cli": ">=0.23.0"
           },
       """
-    And the packages/vendor/wp-cli/foo/command.php file should exist
-    And the packages/vendor/wp-cli/foo/wp-cli.yml file should exist
-    And the packages/vendor/wp-cli/foo/.travis.yml file should not exist
+    And the {PACKAGE_PATH}/local/wp-cli/foo/command.php file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/foo/wp-cli.yml file should exist
+    And the {PACKAGE_PATH}/local/wp-cli/foo/.travis.yml file should not exist
 
-    When I run `WP_CLI_PACKAGES_DIR=packages wp --require=packages/vendor/wp-cli/foo/command.php hello-world`
+    When I run `wp --require={PACKAGE_PATH}/local/wp-cli/foo/command.php hello-world`
```

Comment:
This `--require` statement shouldn't be necessary now, because the command is autoloaded.